web/content/js/nav.js

```diff
@@ -91,10 +91,10 @@ let navItems = [
 		href: "/Quickstart_Guide/index-katello.html"
 	},{
 		title: "Installing Katello Server",
-		href: "/Installing_Server_on_Red_Hat/index-katello.html"
+		href: "/Installing_Server/index-katello.html"
 	},{
 		title: "Installing Smart Proxy with Content",
-		href: "/Installing_Proxy_on_Red_Hat/index-katello.html"
+		href: "/Installing_Proxy/index-katello.html"
 	},{
 		title: "Upgrading and Updating",
 		href: "/Upgrading_and_Updating/index-katello.html"
```

Comment:
Do you plan to backport it down to 2.4? Could experience a conflict or two there.

Reply:
See the initial message: it's split into two commits so you can cleanly pick one (copy the guide) and not the other (remove the old guides).